src/auth.py

```diff
@@ -4,15 +4,15 @@
 from decouple import Csv, config, UndefinedValueError
 from opentaxii.auth.api import OpenTAXIIAuthAPI
 from opentaxii.entities import Account
-import django.contrib.auth
+from stip.common.rest_api_auth import auth_by_api_key
 
 
 class StipTaxiiServerAuth(OpenTAXIIAuthAPI):
     def __init__(self):
         pass
 
     def authenticate(self, username, password):
-        stip_user = django.contrib.auth.authenticate(username=username, password=password)
+        stip_user = auth_by_api_key(username, password)
```

Comment:
Is the variable name "password" instead of "api_key"?

Reply:
Thanks @uchimrina .
I changed the variable name and confirmed that it works the same.
Please review again.